tests/python/relax/test_relax_operators.py

```diff
@@ -193,5 +193,20 @@ def test_op_shape_of():
     assert constrained_shape == tvm.runtime.ShapeTuple([1])
 
 
+@tvm.script.ir_module
+class TensorToShapeTest:
+    @R.function
+    def run_tensor_to_shape(t: R.Tensor(ndim=1, dtype="int64")) -> R.Shape((1, 2, 3)):
+        gv: R.Shape(ndim=3) = R.tensor_to_shape(t)
+        return gv
+
+
+def test_op_tensor_to_shape():
+    out_shape = run_cpu(
+        TensorToShapeTest, "run_tensor_to_shape", tvm.nd.array(np.array([1, 2, 3]).astype("int64"))
```

Comment:
Noting my comment above, I'm not sure this is the proper behavior. The input tensor has `ndim=1`. Should the produced shape really have `ndim=3`?

Reply:
I think this is correct. We are converting the rank-1 tensor [1, 2, 3] into a shape (1, 2, 3). I think the problem here is that `ndim` means different things for tensors (where it means the rank) and shapes (where it means the number of dimensions denoted by the shape).